Caching.Redis/TagCache/RedisClient.cs

```diff
@@ -271,6 +304,9 @@ public Task Expire(string key, DateTime expiry)
         public Task<bool> Exists(string key)
         {
             var connection = _connectionManager.GetConnection();
+            if (connection == null)
+                throw new RedisConnectionException(ConnectionFailureType.InternalFailure, "There was an error when retrieving the Redis Connection.");
```

Comment:
Wondering if you could change GetConnection in _connectionManger to do a null check and throw, rather than every consumer  of the  GetConnection response doing a null check and throw

Reply:
Great catch - I wasn't thinking and just fixed each error as they showed up in Rider!